### ISSUE TITLE
Fix `computeEditInfo` tests for latest Jest preset

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#8508]
When updating to the latest [Jest preset](https://github.com/stylelint/jest-preset-stylelint/releases/tag/7.3.0):

```console
npm i -D jest-preset-stylelint@latest
```

There are failing tests:

```console
Test Suites: 7 failed, 290 passed, 297 total
Tests:       17 failed, 11 skipped, 9892 passed, 9920 total
```

Related to the `fix` object, e.g.:

```
comment-empty-line-before › reject › [ 'always', { except: [ 'first-nested' ] } ] › 'a { color: pink;\r\n/** comment */\r\ntop: 0; }' › CRLF

    expect(received).toMatchObject(expected)

    - Expected  - 1
    + Received  + 9

      Object {
    -   "fix": undefined,
    +   "fix": Object {
    +     "range": Array [
    +       17,
    +       18,
    +     ],
    +     "text": "
    + 
    + ",
    +   },
        "text": "Expected empty line before comment (comment-empty-line-before)",
      }

      at Object.<anonymous> (node_modules/jest-preset-stylelint/getTestRule.js:116:33)
```

If anyone has time and is more familiar with these tests, please open a PR to fix. These are likely the false negatives that the release is picking up.

Blocking https://github.com/stylelint/stylelint/pull/8498

**Comment by ybiquitous** [comment]
Related: https://github.com/stylelint/jest-preset-stylelint/pull/141

**Comment by ybiquitous** [comment]
I've pushed https://github.com/stylelint/stylelint/pull/8510/commits/ccee5b41bb36b39177eae20f3d1a465a493ab119 to fix the failed tests.

**Comment by ybiquitous** [comment]
Now #8510 CI is green ✅
